PharoObjectModel/PharoObjectModel.pier

```diff
@@ -292,56 +296,66 @@ Dog class>>>count
     ^ count
 ]]]
 
```

Comment:
Oh, and it's easier to look at the diffs (and the comments) in the Files Changed tab, not the Conversations tab.